### DISCUSSION TITLE
How to bootstrap the Pypi copr project?

### DISCUSSION THREAD

**Discussion by pkking** [fedora-copr/copr#2446] [General]
Hi there, maybe this is not a good place :)
I notice there's a [pypi project](https://copr.fedorainfracloud.org/coprs/g/copr/PyPI/) on fedora copr, a huge number of pypi packages and builds, and im wondering how this project aims to and how it was bootstrapped(may be just some robust scripts and some time?)

**Comment by FrostyX** [comment]
Hello @pkking,
there is also [@rubygems/rubygems](https://copr.fedorainfracloud.org/coprs/g/rubygems/rubygems/) :-)

We collectively wrote a blog post about both of those projects,
https://developers.redhat.com/articles/2022/06/07/thousands-pypi-and-rubygems-rpms-now-available-rhel-9
Does it answer your questions?

I also wrote a blogpost about RubyGems
http://frostyx.cz/posts/rebuilding-the-entire-rubygems-in-copr
Many of the things that I talk about there are true also for the PyPI project, there are just different numbers and tool names :-)

**Comment by xsuchy** [comment]
Migrated to the discussion.

The bootstrap... There was no big intelligence in the submission. We simply built everything and then again the rest. And again. Well... with PyPI Karolina sorted the packages by a number of download - with the idea that the most downloaded are the most maintained ones. Compared to a module with few download and the module may not even build.

**Comment by pkking** [comment]
Thanks the kind answers, thats all my wanted to know :)
